src/sys/unix/waker.rs

```diff
@@ -265,6 +265,32 @@ mod pipe {
                 }
             }
         }
+
+        #[cfg(target_os = "aix")]
+        fn pipe2() -> io::Result<(RawFd, RawFd)> {
```

Comment:
Since this is essentially the same code as in `src/sys/unix/pipe.rs`, maybe we should reuse that code.